swagger.yaml

```diff
@@ -321,4 +329,7 @@ definitions:
   Limit:
     type: string
     description: "The limit applied to the number of results returned"
+  Offset:
+    type: string
+    description: "The offset applied to the number of results returned"
```

Comment:
It has been standard i our other api's (e.g. dp-dataset-api, dp-search-api) to describe the offset like so:
`The first row of <resources> to retrieve, starting at 0. Use this parameter as a pagination mechanism along with the limit parameter.`

Where the resource could represent not just a single word (like `editions`) but to describe the resources in question, `editions for a code list`